src/main/java/gmail/taras/shevchuk0024/lesson40_Functional_Interface/UnaryOperator.java

```diff
@@ -0,0 +1,16 @@
+package gmail.taras.shevchuk0024.lesson40_Functional_Interface;
+
+public class UnaryOperator {
+    public static java.util.function.UnaryOperator<Integer> fibonacci = n -> {
+        if (n == 0) return 0;
+        if (n == 1) return 1;
+
+        int a = 0, b = 1;
+        for (int i = 2; i <= n; i++) {
+            int temp = a + b;
+            a = b;
+            b = temp;
+        }
+        return b;
+    };
+}
```

Comment:
Додайте ще рекурсивний метод 😉 

```suggestion
package gmail.taras.shevchuk0024.lesson40_Functional_Interface;

import java.util.function.UnaryOperator;

public class UnaryOperatorUtils {
    public static UnaryOperator<Integer> fibonacci = n -> {
        if (n == 0) {
            return 0;
        }
        
        if (n == 1) {
            return 1;
        }

        int a = 0;
        int b = 1;

        for (int i = 2; i <= n; i++) {
            int temp = a + b;
            a = b;
            b = temp;
        }

        return b;
    };
}
```